pyspectrafuse/consensus_strategy/average_spectrum_strategy.py

```
@@ -23,17 +46,52 @@ def __init__(self, DIFF_THRESH=0.01, DYN_RANGE=1000, MIN_FRACTION=0.5, pepmass='
+        Args:
+            cluster_df: DataFrame containing cluster spectrum data
+            filter_metrics: Metric column name for filtering (unused in this strategy)
+            
+        Returns:
+            Tuple of (consensus_spectrum_df, single_spectrum_df)
+            
+        Raises:
+            ValueError: If cluster_df is empty
+        """
+        if cluster_df.empty:
+            raise ValueError("cluster_df is empty")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)